crates/bounce/src/root_state.rs

```diff
@@ -76,7 +76,7 @@ impl BounceRootState {
     where
         T: 'static,
     {
-        let notion_states = self.notion_states.borrow();
+        let notion_states = self.notion_states.borrow().clone();
```

Comment:
If `notion_states.get()` is performed before clone, then you should be able to only clone the states of a single notion.

Reply:
But you have to borrow it to get anything from it? Hence the PR.

Reply:
I think it should be possible to:

```suggestion
        let notion_states = self.notion_states.borrow().get(&TypeId::of::<T>()).clone();
```